README.md

```diff
@@ -176,6 +182,7 @@ Description:
 
 ## Changelog
 
+- 0.2.2 - Add support for ns records
```

Comment:
Don't know how versioning works for this repo, but this adds a feature, it should be 
```suggestion
- 0.3.0 - Add support for NS records
```

Reply:
Hi, To be clear, there are two types of terraform code: the `master` branch for the newer terraform version ( 0.12 ) and the old one - `terraform011` branch for terraform 0.11 version, therefor the tagging 0.3.x is only for terraform 0.12 and 0.2.x is only for terraform 0.11

Reply:
It depends on what version of your terraform version you are currently working on, It would be nice if you create another PR to `terraform011` branch in which coding terraform 0.11 style,
since our infrastructure as code haven't upgraded yet

Reply:
`0.2.2` should a target tagging version

Reply:
Hey @phuong-leeo, thanks for pointing out that there is a `terraform011` branch, this can easily be missed! IMHO it should be documented in the README.

I don't see how this implies the version number, though. It looks like SemVer and we are adding a feature. If the change from 0.1 to 0.2 was a breaking one (because Terraform 0.12 was used and is incompatible with 0.11), we are using SemVer wrong. Any weird mapping between Terraform versions and verions of this repo a) are questionable because of SemVer confusions and b) should be documented.

Reply:
sadly, We do a bit diverge tagging ( wrong SemVer tagging) 

```
0.1.xx -> terraform 0.11
0.2.x -> terraform 0.12
```
Could we agree? As mostly out TF modules did the same way

Reply:
@phuong-leeo I guess your statement is not quite right, i.e. [here ](https://github.com/ryte/INF-tf-rds) and [there](https://github.com/ryte/INF-tf-s3) a classic SemVer is used and the current version is 0.4.0, so I guess only the modules migrated to terraform 0.12 and without added new features are still on the version 0.2.x

Reply:
I'm no DevOp, it's not up to me to decide, so I can live with that.
It's just that I advise against it, because every dev who is not familiar with this like me is likely to make the same mistake / assume that 0.2 is compatible with 0.1 etc.

Reply:
okay, we should keep it as is